apps/acceptance/clock/acceptanceStopwatch.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
+var Clock = require('./regions/appView');
+var expect = require('chai').expect;
+
+marionette('Clock Acceptance Suite', function(){
+    var client;
+    var clock;
+    client = marionette.client({
+        prefs: {
+            // we need to disable the keyboard
+            // to avoid intermittent failures on
+            // Travis (transitions might take
```

Comment:
Travis is not used anymore. I don't think that comment is valid anymore. The restriction might still be, though